package.json

```diff
@@ -21,7 +21,7 @@
     "@angular/platform-server": "~17.0.0",
     "@angular/router": "~17.0.0",
     "@angular/ssr": "~17.0.0",
-    "@ngrx/signals": "^17.1.0",
+    "@ngrx/signals": "^17.2.0",
```

Comment:
Is that mandatory?

Reply:
Yes I had some type problems with the withEntity when I used 17.1.0